src/ddspdrum/parameter.py

```diff
@@ -57,7 +65,7 @@ def set_value(self, new_value):
         ---------
         new_value (float)   :   value to update parameter with
         """
-        self.value = np.clip(new_value, self.minimum, self.maximum)
+        self.value = new_value
```

Comment:
Should we assert here as well?

Reply:
yes, everywhere there's a clip it indicates a precondition to assert, IMHO